doc/Overview/Reactive/in-apps.md

```diff
@@ -120,3 +119,53 @@ This will be exposed into an `ICommand` that can be data-bound to the `Command`
 ```xml
 <Button Command="{Binding Share}" Content="Share" />
 ```
+
+By default, if the method has a parameter `T myValue` and there is a property `Feed<T> MyValue` on the same class (matching type and name), 
+that parameter will be automatically full-filed from that feed.
+
+For instance, in your ViewModel:
+
+```csharp
+public IFeed<string> Message { get; }
+
+public async ValueTask Share(string message) 
+{
+
+}
+```
+
+Can be used with or without any `CommandParameter` from the view:
+
+```xml
+<!-- Without CommandParameter -->
+<!-- 'message' arg in the 'Share' method will be the current value of the Message _feed_ -->
+<Button Command="{Binding Share}" Content="Share" />
+
+<!-- With CommandParameter -->
+<!-- 'message' arg in the 'Share' method will be "hello world" -->
+<Button Command="{Binding Share}" CommandParameter="hello world" Content="Share" />
+```
+
+You can even mix "feed parameters" with the "view parameter" (i.e. the value of the `CommandParameter`):
+
+```csharp
+public IFeed<MyEntity> Entity { get; }
+
+public async ValueTask Share(MyEntity myEntity, string origin) 
```

Comment:
Why isn't the parameter `entity`? How does the matching work?

Reply:
Humm, indeed the parameter should be `entity`. Matching works on type and name (explained above)

```suggestion
public async ValueTask Share(MyEntity entity, string origin) 
```